src/moments.jl

```diff
@@ -160,39 +270,39 @@ function _moment3(v::RealArray, m::Real)
         @inbounds z = v[i] - m
         s += z * z * z
     end
-    s / n
+    varcorrection(n, false) * s
```

Comment:
Yes, I'd prefer that. I don't see the point in calling a function which is known to be incorrect just to get `1/n`.